pantheon-bundle/frontend/spandx.config.js

```diff
@@ -0,0 +1,47 @@
+require("dotenv").config();
+
+// If we have a .env file respect the HOST and PORT settings
+// If not default to localhost
+const HOST = process.env.HOST || 'localhost';
+// If not default to 9595
+const PORT = process.env.PORT || 9595;
+
+module.exports = {
+  host: {
+    local: HOST
+  },
+  port: PORT,
+  open: true,
+  startPath: "/",
+  verbose: true,
+  routes: {
+    // Here are some routing examples to get started.
+    "/": {
+      host: {
+          local: "http://localhost:9000"
+      },
+      path: "/"
+    },
+
+    "/content": {
+      host: {
+        local: "http://localhost:8181"
+      }
+    },
+    "/system": {
+      host: {
+        local: "http://localhost:8181"
+      }
+    },
+    "/pantheon": {
+      host: {
+        local: "http://localhost:8181"
+      }
+    }
```

Comment:
One thing I noticed while I was playing with this is this - while the application 99% works in "hot frontend" mode, the login screen does not. The login request results in a 404. I'm wondering if we need to add an additional route here to account for the login api endpoint?

Here's a screenshot that might help:
![image](https://user-images.githubusercontent.com/22244893/88817705-d9071200-d18b-11ea-9746-6db207223832.png)

Reply:
Ahh, yes! I can capture that login request to make sure it goes to Karaf.

Good catch!